BDArmory/CounterMeasure/CMDropper.cs

```diff
@@ -92,21 +96,28 @@ public override void OnStart(StartState state)
                 SetupCM();
 
                 ejectTransform = part.FindModelTransform(ejectTransformName);
+                if (ejectTransform == null)
+                    ejectTransform = part.transform;
 
                 if (effectsTransformName != string.Empty)
                 {
                     effectsTransform = part.FindModelTransform(effectsTransformName);
                 }
 
-                part.force_activate();
+                if (part.FindModuleImplementing<MissileLauncher>() != null)
```

Comment:
`MissileBase` might be better to also catch those on modular missiles and then similar CM code could be added to those.
Also in ModuleECMJammer.cs.

Reply:
Sorry, been busy all day. The modular missile issue is much more difficult to deal with, unless you have the missile controller also be the part with the CMDropper/jammer we'd have to recognize that a certain CMDropper/jammer is part of the modular missile sub-craft. If we want to modular missiles to have a similar set of features it'd be a completely different piece of code, we'd probably have to have the `isMissile` field be a KSP field and allow people to introduce parts for that specific purpose, and then have the modular missile perform a scan for CMDroppers and jammers post-launch.

Reply:
Ah right, yeah I didn't think that all the way through. However, you could use `SpawnUtils.IsModularMissilePart` to check if they're part of a modular missile.
If the rest of the code would need major changes to make it work with them, then it's probably not worth it for now, but I suspect you'll get requests to make it work for modular missiles too once people see it as an option for regular missiles.